pallets/loans/src/market.rs

```diff
@@ -0,0 +1,23 @@
+use crate::InterestRateModel;
+use primitives::{Rate, Ratio};
+
+/// The current state of a market. For more information, see [Market].
+#[derive(codec::Decode, codec::Encode, sp_runtime::RuntimeDebug)]
+pub enum MarketState {
+    Active,
+    Pending,
+    Supervision,
+}
+
+#[derive(codec::Decode, codec::Encode, sp_runtime::RuntimeDebug)]
+/// Market.
+///
+/// A large pool of liquidity where accounts can lend and borrow.
+pub struct Market {
+    close_factor: Ratio,
+    collateral_factor: Ratio,
+    liquidate_incentive: Rate,
+    market_state: MarketState,
+    rate_model: InterestRateModel,
+    reserve_factor: Ratio,
```

Comment:
I suggest the sequence of the fields:
collateral_factor
reserve_factor
close_factor
liquidate_incentive
rate_mode
state

Reply:
Fields were re-arranged